libraries/service/src/main/AndroidManifest.xml

```diff
@@ -3,12 +3,23 @@
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE_DATA_SYNC"/>
   <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
+  <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
 
   <application tools:node="merge">
     <service
         android:name="org.holochain.androidserviceruntime.service.HolochainService"
         android:foregroundServiceType="dataSync"
         android:exported="true">
     </service>
+    
+    <receiver
+        android:name="org.holochain.androidserviceruntime.service.BootCompletedReceiver"
```

Comment:
Wires up the android system to send the `BOOT_COMPLETED` intent to our `BootCompletedReceiver` on boot.